modules/ROOT/pages/producing-ign.adoc

```diff
@@ -92,6 +93,25 @@ curl https://getfedora.org/static/fedora.gpg | gpg --import
 gpg --verify butane-x86_64-unknown-linux-gnu.asc
 ----
 
+==== macOS
+To use the Butane binary on macOS, follow these steps:
+
+. If you have not already done so, download the https://getfedora.org/static/fedora.gpg[Fedora signing keys] and import them:
++
+[source,bash]
+----
+curl https://getfedora.org/static/fedora.gpg | gpg --import
+----
+. Download the latest version of Butane and the detached signature from the https://github.com/coreos/butane/releases[releases page].
+. Verify it with gpg:
++
+[source,bash]
+----
+gpg --verify butane-x86_64-apple-darwin.asc
+----
+
+Alternatively, you can install Butane via https://brew.sh[Homebrew]: `brew install butane`
```

Comment:
For consistency, let's move this command into a code block also.

Reply:
Should I break the sections up into `===== Standalone` and `===== Homebrew` for macOS as well then? Someone might add a MacPorts version ;-)

Reply:
Actually, you're right: we already have a distro package section, so this doesn't belong here.  Let's generalize the `Installing via Fedora packages` section to e.g. `Installing via distribution packages` and have subsections for Fedora, Homebrew, etc.

Reply:
Will update the document with your suggestion in a short while 👍

Reply:
> Someone might add a MacPorts version ;-)

So... I did.  :grin:  Docs update in https://github.com/coreos/fedora-coreos-docs/pull/275.